docs/kb/ssl.md

```diff
@@ -57,149 +57,180 @@ Prior to V4.1t 2022.03.25, kdb+ would load the following files:
-$ export SSL_CERT_FILE=$HOME/certs/server-crt.pem
-$ export SSL_KEY_FILE=$HOME/certs/server-key.pem
-```
+Configuration of keys/certificates and checks performed are controlled by the following [environment variables](https://en.wikipedia.org/wiki/Environment_variable)
 
 !!! info "KX first"
 
-    Since V3.6, kdb+ gives preference to the `KX_` prefix for the `SSL_*` environment variables to avoid clashes with other OpenSSL based products. 
+    Since V3.6, kdb+ gives preference to the `KX_` prefix for the `SSL_*` environment variables to avoid clashes with other OpenSSL based products.
 
     For example, the value for `` getenv`KX_SSL_CERT_FILE`` has a higher precedence than `` getenv`SSL_CERT_FILE`` for determining config.
 
-If you don’t have a certificate, you can create a self-signed certificate using the `openssl` program. An example script to do so follows, which you should customize as necessary
+#### SSL_CERT_FILE
 
-```bash
-$ more makeCerts.sh
-mkdir -f $HOME/certs && cd $HOME/certs
+The certificate file. This must be in PEM format and must be sorted starting with the subject's certificate (actual client or server certificate), followed by intermediate CA certificates if applicable, and ending at the highest level (root).
+
+Default value is `<OPENSSLDIR>/server-crt.pem`
+
+#### SSL_CA_CERT_FILE
 
-# Create CA certificate
-openssl genrsa 2048 > ca-key.pem
-openssl req -new -x509 -nodes -days 3600 \
-    -key ca-key.pem -out ca.pem -extensions usr_cert \
-    -subj '/C=US/ST=New York/L=Brooklyn/O=Example Brooklyn Company/CN=examplebrooklyn.com'
-
-# Create server certificate, remove passphrase, and sign it
-# server-crt.pem = public key, server-key.pem = private key
-openssl req -newkey rsa:2048 -days 3600 -nodes \
-    -keyout server-key.pem -out server-req.pem -extensions usr_cert \
-    -subj '/C=US/ST=New York/L=Brooklyn/O=Example Brooklyn Company/CN=myname.com'
-openssl rsa -in server-key.pem -out server-key.pem
-openssl x509 -req -in server-req.pem -days 3600 -CA ca.pem -CAkey ca-key.pem \
-    -set_serial 01 -out server-crt.pem -extensions usr_cert
-
-# Create client certificate, remove passphrase, and sign it
-# client-crt.pem = public key, client-key.pem = private key
-openssl req -newkey rsa:2048 -days 3600  -nodes \
-    -keyout client-key.pem -out client-req.pem -extensions usr_cert \
-    -subj '/C=US/ST=New York/L=Brooklyn/O=Example Brooklyn Company/CN=myname.com'
-openssl rsa -in client-key.pem -out client-key.pem
-openssl x509 -req -in client-req.pem -days 3600 -CA ca.pem -CAkey ca-key.pem \
-    -set_serial 01 -out client-crt.pem -extensions usr_cert
+A file of certificate authority (CA) certificates in PEM format. The file can contain several CA certificates identified by
+
+```
+-----BEGIN CERTIFICATE-----
+... (CA certificate in base64 encoding) ...
+-----END CERTIFICATE-----
 ```
+sequences. Before, between, and after the certificates text is allowed which can be used e.g. for descriptions of the certificates.
```

Comment:
```suggestion
sequences. Text is allowed before, between, and after the certificates; it can be used, for example, for descriptions of the certificates.
```